test/e2e/rule_test.go

```diff
@@ -556,6 +560,141 @@ func TestRule(t *testing.T) {
 	}
 }
 
+func TestRule_CanRemoteWriteData(t *testing.T) {
+	t.Parallel()
+
+	testAlertRuleRecordAbsentMetric := `
+groups:
+- name: example_record_rules
+  interval: 100ms
+  rules:
```

Comment:
Let's also add Alert so we have both types (:  and this is when variable name makes sense ;p